wikipedia.vislcg.txt

```diff
@@ -168,227 +170,228 @@
 	"." CLB @punct #12->5
 
 "<Hon>"
-	"hon" Pron Pers Sg3 Fem Nom @nsubj #1->1
+	"hon" Pron Pers Sg3 Fem Nom @nsubj:cop #1->5
 "<er>"
-	"vera" V Ind Prs Sg3 @cop #2->2
+	"vera" V Ind Prs Sg3 @cop #2->5
 "<eisini>"
-	"eisini" Adv @dep #3->3
+	"eisini" Adv @advmod #3->2
```

Comment:
Hmm, I guess that makes more sense. If I try to break it down, and remove words while keeping it intact, it might make sense to say "It is also the haven" which I guess makes it clearer to me. I don't know if it makes sense to think like that.